### PR TITLE
feat: Add support for Arbitrum Goerli

### PR DESCRIPTION
**Pull request by luzzif**
# Summary

The PR adds support to Arbitrum Goerli. Currently a WIP since to be merged it needs [this PR](https://github.com/levelkdev/swapr-sdk/pull/175) to be merged, a release of the SDK and then an SDK upgrade.

# To Test

Pretty much everything that has to do with contracts interactions.

- Swapping
- Adding/removing liquidity
- Creating liquidity mining campaigns
- Staking/unstaking/claiming rewards
- Everything else if I'm missing something...

Bridging support I haven't had a look at so it might need changes/fixes. 

Tokens needed to perform tests can be merged from Goerli [here](https://bridge.arbitrum.io/).

### CODE REVIEW

File: src/apollo/client.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const subgraphClientsUris: { [chainId in SWPRSupportedChains]: string } =
   [ChainId.XDAI]: 'https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-xdai-v2',
   [ChainId.RINKEBY]: 'https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-rinkeby',
   [ChainId.ARBITRUM_RINKEBY]: 'https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-arbitrum-rinkeby-v2',
+  [ChainId.ARBITRUM_GOERLI]: '', // FIXME: fix this once the subgraph is deployed
```
Comment (adamazad) [inline src/apollo/client.ts:14]: There's no support for Arbitrum Görli for now.

File: src/lib/eco-router/api.ts [hunk 2]
```diff
@@ -177,6 +177,7 @@ export async function getExactIn(
       currencyAmountIn,
       currencyOut,
       maximumSlippage,
+      user: '', // FIXME: is this right? */
```
[the remark this answers is not in the document]
Comment (karczuRF) [reply src/lib/eco-router/api.ts:180]: Ok gotcha. Thanks for explaining ;)

**Comment by niemam29** [comment]
Not sure if it's ready for testing but for now it shows that goerli is unsupported

cache3

**Comment by niemam29** [comment]
There is a problem with icon
cache3

File: src/components/Header/index.tsx [hunk 3]
```diff
@@ -251,7 +251,14 @@ function Header() {
...
             {t('swap')}
           </HeaderLink>
-          <HeaderLink data-testid="pool-nav-link" id="pool-nav-link" to="/pools" disabled={networkWithoutSWPR}>
+          <HeaderLink
+            data-testid="pool-nav-link"
+            id="pool-nav-link"
+            to="/pools"
+            disabled={
+              networkWithoutSWPR && chainId !== ChainId.ARBITRUM_GOERLI
+            } /* // FIXME: fix this once SWPR is on Arb Goerli */
```
Comment (berteotti) [inline src/components/Header/index.tsx:260]: Shouldn't we update this value inside `SWPR[chainId]`?

Comment (adamazad) [reply src/components/Header/index.tsx:260]: @luzzif, do we have Swapr on Arbitrum Görli yet?

**Comment by MilanVojnovic95** [comment]
Tested and the icon thing found by Kacper has been fixed the only thing i have successfully tested is wrapping ETH to WETH because other tokens are not available

image
image

Some pages have a 'NOT AVAILABLE' badge, although for example the 'Liquidity' page is only clickable, but endless loading occurs.

image
image

**Comment by Mi-Lan** [comment]
> Tested and the icon thing found by Kacper has been fixed the only thing i have successfully tested is wrapping ETH to WETH because other tokens are not available
> 
> image image
> 
> Some pages have a 'NOT AVAILABLE' badge, although for example the 'Liquidity' page is only clickable, but endless loading occurs.
> 
> image image

I also noticed this my suggestion so we push release forward is to just not display it as option for now in the network picker like for rinkeby and to polish it in next release. So we can merge it and test cowtrade pr.... @adamazad wdyt?

**Comment by adamazad** [comment]
> Tested and the icon thing found by Kacper has been fixed the only thing i have successfully tested is wrapping ETH to WETH because other tokens are not available
> 
> image image

This is OK. The Arbitrum Görli new is new. We don't have an official token list yet. 

On swapping, there are no pairs yet AFAIK. 

> Some pages have a 'NOT AVAILABLE' badge, although for example the 'Liquidity' page is only clickable, but endless loading occurs.
> 
> image image

This is an issue on The Graph's side. I'm still waiting for an update from the team:

image

**Review by Mi-Lan — APPROVED** [review]
As I mentioned in previous comment in an effort to push this release forward I think we should  merge it and fix these issues in next release. I think it would be good to remove it from network picker for now since its unusable.
Screenshot 2022-08-05 at 14 16 00

**Pull request merged**